modules/javafx.web/src/main/java/com/sun/webkit/network/CookieJar.java

```diff
@@ -40,7 +42,9 @@ private CookieJar() {
     }
 
     private static void fwkPut(String url, String cookie) {
-        CookieHandler handler = CookieHandler.getDefault();
+        @SuppressWarnings("removal")
+        CookieHandler handler =
+            AccessController.doPrivileged((PrivilegedAction<CookieHandler>) () -> CookieHandler.getDefault());
```

Comment:
Not sure if there is a policy for this, but you could replace the lambda with the method reference?

Reply:
Good idea. Done.